js/popup.js

```diff
@@ -596,9 +596,9 @@ async function scrapeHtml(url, urlDepth) {
       try {
         // Download various resources from the webpage
         html = await getJavaScript(html, url, urlDepth); // Download external JavaScript files
-        html = await getCSS(html, url, urlDepth); // Download CSS files
+        html = await getCSS(html, url, urlDepth); // Download CSS file
         // Download images if the user has not opted to exclude them
-        if (!isExcludeImages) {
+        if (!isFocusMode) {
           html = await getImgs(html, url, urlDepth);
         }
```

Comment:
This makes sense. Also, now I can make sense of your above changes. We are basically not needing the check for exclude images because it is now handled through focus mode. Now thinking about this we should probably remove the exclude images entirely. Again, it is important to do changes within one related scope and small changes if possible, you seem to have two scopes here: one being focus mode the other being ui updates.

Reply:
okay, I will remove ExcludeImage entirely